### PR TITLE
Updated support for better account generation

### LINKED ISSUE #90: Add chain-id to generate-mnemonic command
Currently theres no way to specify account creation for accounts other than `juno` [see here](https://github.com/CronCats/croncat-rs/blob/main/croncat/src/store/agent.rs#L142).

Need to add `chain-id` so the bech32 account generates for the right network.

### PR DESCRIPTION
**Pull request by TrevorJTClarke**
Fixes #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +69,11 @@ cargo doc --no-deps
 - Add agent into local keystore
 
 ```bash 
+## Generate for a specific network
+## "prefix" is the Bech32 chain prefix
+cargo run generate-mnemonic --new-name juno --prefix juno
+
+## If you really like insecure setup:
```
Comment (mikedotexe) [inline README.md]:
```suggestion
```

Comment (mikedotexe) [reply README.md]: Don't think it's useful to mock our end users in a README

**Comment by mikedotexe** [comment]
Screen Shot 2022-10-26 at 12 11 29 PM

I thought that might be because we have this in the `config.local.yaml`:

https://github.com/CronCats/croncat-rs/blob/f69d6912041468c6f2b540023a3488edddaabafa/config.local.yaml#L1-L2

But after changing that to:

```diff
- denom: "ujunox"
+ denom: "stake"
- prefix: "juno"
+ prefix: "local"
chain_id: "testing"
rpc_endpoint: "http://localhost:26657/"
grpc_endpoint: "http://localhost:9090/"
wsrpc_endpoint: "ws://localhost:26657/websocket"
contract_address: null # This can be set via CRONCAT_CONTRACT_ADDRESS, committing this is a pain.
gas_prices: 0.07
gas_adjustment: 1.5
polling_duration_secs: 2
```

am still seeing this

File: croncat/src/store/agent.rs [hunk 2]
```diff
@@ -181,20 +170,46 @@ impl LocalAgentStorage {
         );
     }
 
+    pub fn display_addrs(&self, account_id: &str) {
```
Comment (TrevorJTClarke) [inline croncat/src/store/agent.rs]: @SeedyROM can you look at this? I need halps making this work

**Comment by TrevorJTClarke** [comment]
Screen Shot 2022-10-26 at 3 30 51 PM

Added new cmd: 
`cargo run -- get-agent-accounts --sender-name mainnet` - gets all generated addrs for keypair
`cargo run -- get-agent-accounts --sender-name mainnet --chain-id juno-1` - gets addr for network

**Comment by TrevorJTClarke** [comment]
Screen Shot 2022-10-26 at 3 33 59 PM

Now upon generating new account, spits out supported addresses for easier funding

**Comment by TrevorJTClarke** [comment]
Hmm found an issue now:
```
status: Unknown, message: "empty address string is not allowed", details: [], metadata: MetadataMap { headers: {"content-type": "application/grpc", "x-cosmos-block-height": "5411507"} }
```

`croncat-rs/croncat/src/client/query_handlers/wasm_query.rs:29:19`

**Pull request merged**